### PR TITLE
profiling storage detail

### PR DESCRIPTION
**Pull request by cangfengzhs**
image

### CODE REVIEW

**Review by critical27 — COMMENTED** [review]
Generally LGTM
Add the detail to IndexScanExecutor as well.
Use camel style naming.

**Review by liuyu85cn — APPROVED** [review]
LGTM

**Review by critical27 — APPROVED** [review]
LGTM, good job

File: src/storage/exec/AggregateNode.h [hunk 1]
```diff
@@ -35,13 +35,15 @@ struct PropStat {
 template <typename T>
 class AggregateNode : public IterateNode<T> {
  public:
-  using RelNode<T>::execute;
+  using RelNode<T>::doExecute;
 
   AggregateNode(RuntimeContext* context, IterateNode<T>* upstream, EdgeContext* edgeContext)
-      : IterateNode<T>(upstream), context_(context), edgeContext_(edgeContext) {}
+      : IterateNode<T>(upstream), context_(context), edgeContext_(edgeContext) {
+    IterateNode<T>::name_ = "AggregateNode";
```
Comment (yixinglu) [inline src/storage/exec/AggregateNode.h:42]: @cangfengzhs 

I really don't recommend this way to initialize the fields of the base class. Please provide the constructor in the base class to do this, such as:

```c++
IterateNode<T>::IterateNode<T>(const std::string& name) : name_(name) {}

AggregateNode(RuntimeContext* context, IterateNode<T>* upstream, EdgeContext* edgeContext) 
  : IterateNode("AggregateNode", upstream), context_(context)...{}
```

File: src/storage/exec/EdgeNode.h [hunk 2]
```diff
@@ -71,15 +74,17 @@ class EdgeNode : public IterateNode<T> {
...
 
   FetchEdgeNode(RuntimeContext* context,
                 EdgeContext* edgeContext,
                 EdgeType edgeType,
                 const std::vector<PropContext>* props,
                 StorageExpressionContext* expCtx = nullptr,
                 Expression* exp = nullptr)
-      : EdgeNode(context, edgeContext, edgeType, props, expCtx, exp) {}
+      : EdgeNode(context, edgeContext, edgeType, props, expCtx, exp) {
+    name_ = "FetchEdgeNode";
```
Comment (yixinglu) [inline src/storage/exec/EdgeNode.h:86]: ditto

File: src/storage/exec/IndexOutputNode.h [hunk 3]
```diff
@@ -44,18 +44,21 @@ class IndexOutputNode final : public RelNode<T> {
         fields_(fields) {
     type_ = context_->isEdge() ? IndexResultType::kEdgeFromIndexScan
                                : IndexResultType::kVertexFromIndexScan;
+    RelNode<T>::name_ = "IndexOpuputNode";
```
Comment (yixinglu) [inline src/storage/exec/IndexOutputNode.h:47]: ditto

**Review by yixinglu — COMMENTED** [review]
Please create a new PR to improve this

**Pull request merged**